docs/source/howto/user_interface.rst

```diff
@@ -38,6 +25,27 @@ or folder.
 To learn more about URLs in JupyterLab and Jupyter Notebook, visit
 `starting JupyterLab <http://jupyterlab.readthedocs.io/en/latest/getting_started/starting.html>`_.
 
+Classic Notebook
+================
```

Comment:
```suggestion
Classic Notebook Interface
==========================
```

just to disambiguate a bit, since "Notebook" is already such an overloaded term.